### PR TITLE
update pehar rara handling for transliteration

### PR DESCRIPTION
**Pull request by tarunsingh5**
update english for better pehar rara handling

### CODE REVIEW

**Comment by ManjotS** [comment]
Perfect! Thats exactly where the change would go as it should be treated like M,N

Not sure if theres M,N handling later that might also need R

**Review by navdeepsinghkhalsa — DISMISSED** [review]
Thank you for submitting this fix, Singh Jee!
Unfortunately, this breaks other expected transliteration with respect to ੍ਰ and ਰ
Expected value to be:
`har prabh teriaa`
Received:
`har prbh teriaa`

Expected value to be:
`laal ra(n)g tis kau lagaa`
Received:
`laal r(n)g tis kau lagaa`

**Comment by tarunsingh5** [comment]
@navdeepsinghkhalsa apparently coverage decreased

File: src/__tests__/translit.test.js [hunk 1]
```diff
@@ -13,6 +14,22 @@ describe('translit', () => {
...
+  });
+  it('Should return only english transliteration of Gurmukhi', () => {
+    expect(JSON.stringify(translit('jb lgu drsu n hoie ikau AMimRqu pIvIAY rwm ]')))
+      .toBe('"jab lag dharas na hoi kiau a(n)mrit peeveeaai raam ||"');
+  });
+  it('Should return only english transliteration of Gurmukhi', () => {
+    expect(JSON.stringify(translit('hir hir nwmu iDAweIAY ijs nau ikrpw kry rjwie ]3]')))
+      .toBe('"har har naam dhiaaieeaai jis nau kirapaa kare rajai ||3||"');
+  });
+  it('Should return only english transliteration of Gurmukhi', () => {
```
Comment (tsingh777) [inline src/__tests__/translit.test.js]: nit picking, but every `it` test case should have a more meaningful and unique name. each one of these have `Should return only english transliteration of Gurmukhi` and doesn't really convey what is being tested.

**Review by tsingh777 — COMMENTED** [review]
nice one! make sure to squash and merge when ready so the commit history can stay relatively clean.

File: src/__tests__/translit.test.js (hunk 1 above)
Comment (tarunsingh5) [reply src/__tests__/translit.test.js]: updated

**Review by tsingh777 — APPROVED** [review]
lgtm code wise.

File: src/__tests__/translit.test.js [hunk 2]
```diff
@@ -9,10 +10,26 @@ describe('translit', () => {
         ipa: 'ɕəŋt̪. d͡ʒɪo d͡ʒɑnəh t̪ɪo rɑkʰ hər pɹəɓ t̪erɪəɑ. ket̪e Gəno əsəŋkʰ əʋəGəɳ merɪəɑ. əsəŋkʰ əʋəGəɳ kʰət̪e fere nɪt̪əpɹət̪ səd̪ ɓuliæ. mɔh məGən bɪkərɑl mɑɪɑ t̪o pɹəsɑd̪i Gʰuliæ. luk kərət̪ bɪkɑr bɪkʰəɽe pɹəɓ ner hu t̪e nerɪəɑ. bɪnəʋəŋt̪ nɑnək d̪ɪɑ t̪ɑrəh kɑʈə̀ ɓəʋəd͡ʒəl ferɪəɑ.1.',
       }));
   });
-  it('Should return only english transliteration of Gurmukhi', () => {
+  it('Base test for english transliteration of Gurmukhi', () => {
     expect(JSON.stringify(translit('lwl rMgu iqs kau lgw ijs ky vfBwgw ] mYlw kdy n hoveI nh lwgY dwgw ]1]')))
       .toBe('"laal ra(n)g tis kau lagaa jis ke vaddabhaagaa || mailaa kadhe na hoviee neh laagai dhaagaa ||1||"');
   });
+  it('Test to ensure sihari + rara english transliteration of Gurmukhi is correct', () => {
```
Comment (navdeepsinghkhalsa) [inline src/__tests__/translit.test.js]: You can have multiple `expect().toBe()` under one `it()`

Comment (navdeepsinghkhalsa) [reply src/__tests__/translit.test.js]: Could you please group your new tests under one?

**Review by navdeepsinghkhalsa — APPROVED** [review]

**Pull request merged**